src/js/controller.js

```diff
@@ -470,13 +470,29 @@ export const MediaUIStates = {
       return getShowingSubtitleTracks(controller).map(({ kind, label, language }) => ({ kind, label, language }));
     },
     mediaEvents: ['loadstart'],
-    trackListEvents: ['addtrack', 'removetrack', 'change'],
+    textTracksEvents: ['addtrack', 'removetrack', 'change'],
+  },
+  MEDIA_RENDITION_LIST: {
+    get: function (controller) {
+      const { media } = controller;
+      return [...media?.videoRenditions ?? []];
+    },
+    mediaEvents: ['emptied'],
+    videoRenditionsEvents: ['addrendition', 'removerendition'],
+  },
+  MEDIA_RENDITION_SELECTED: {
+    get: function (controller) {
+      const { media } = controller;
+      return media?.videoRenditions?.[media.videoRenditions?.selectedIndex];
```

Comment:
Nice, the API turned out relatively simple, even without IDs.

I think this property should just return the index value rather than the whole rendition object. As is it's duplicate info with the list, and using just the index means it could still be a simple HTML attribute, even if the list is more complex.

Reply:
after implementing this I remembered why I'd prefer also the complex object here. 
because often (and I would like to use) the quality value to style the rendition / quality button. 
see screenshot

![SCR-20230725-mod](https://github.com/muxinc/media-chrome/assets/360826/e5cb03d5-44ea-4417-8943-fc14a7e60185)


by adding the width and height of the selected rendition we can add HD, 4K labels or icons easily to the button with CSS...

Reply:
To the button itself, not the options? What would that look like exactly? And can the CSS be used reliably with varying widths and heights?

Reply:
good points, I also just reminded it would work better with also having the quality that is actually playing.

something like this

![SCR-20230725-qod](https://github.com/muxinc/media-chrome/assets/360826/71dd12c1-bef3-44e8-829f-4fc87feacb5d)

Reply:
going with rendition index only the `mediarenditionselected` attribute